src/character/CharacterHandler.cpp

```diff
@@ -175,13 +175,33 @@ void CharacterHandler::do_create(std::uint32_t account_id, std::uint32_t realm_i
 	if(items != dbc_.char_start_outfit.end()) {
 		populate_items(character, items->second);
 	} else { // could be intentional, so we'll keep going
-		LOG_DEBUG(logger_) << "No starting item data found for race" <<
+		LOG_DEBUG(logger_) << "No starting item data found for race " <<
 			character.race << ", class " << character.class_ << LOG_ASYNC;

```

Comment:
Existing code but it'd be good to print the class/race as string inside. You can find other examples of that in this file (name.en_gb etc).

Reply:
I thought it was already doing that with the << character.race / << character.class_ - I will look into some of the other examples and alter it accordingly.

Reply:
Looking at the name.en_gb code I'm pretty sure that's what's already going on here as well.